cfgov/jinja2/v1/_includes/organisms/email-signup.html

```diff
@@ -20,17 +20,45 @@
 
 {% if not value %}
     {% set lipsumText = lipsum(1, false, 10, 25) | safe %}
-    {% do global_dict.update(
-        {
-            'email_signup': {
-                  'heading':    'Slug Title',
-                  'text': lipsumText
+    {% macro _privacy_statement() %}
+        The information you provide will permit the Consumer Financial
+        Protection Bureau to process your request or inquiry.
+        <a href="/sub-pages/privacy-policy/">See More</a>
+    {% endmacro %}
+
+    {% macro _form_field() %}
+        {% do global_dict.update({
+            'form_field': {
+                'label':       'Email Address',
+                'name':        'o-email-signup_email',
+                'placeholder': 'example@mail.com',
+                'type':        'email',
+                'info':        {
+                    'source': _privacy_statement()
+                },
+                'required':    true,
+                'btn_text':    'Sign up'
             }
-        })
-    %}
+        }) %}
+        {% set value = global_dict.form_field %}
+        {% include 'molecules/form-field-with-button.html' %}
+    {% endmacro %}
+
+    {% do global_dict.update({
+        'email_signup': {
+            'heading':    'Slug Title',
+            'text':       {
+                'source': lipsumText
+            },
+            'form_field': [
+                _form_field()
+            ]
+        }
+    }) %}
     {% set value = global_dict.email_signup %}
     {% set prototype = true %}
```

Comment:
After testing it seems that I was mistaken and the variable set in an if block will carry past the block as long as the statement is executed. Sorry for the confusion!